content/docs/5.1/concepts/overview.md

```diff
@@ -1,9 +1,38 @@
 ---
 title: Get Started
-description: TODO
+description: TiKV Introduction
 menu:
     "5.1":
         weight: 1
 ---
 
-TODO
+**TiKV** is a highly scalable, low latency, and easy to use key-value database that delivers single-digit millisecond performance at any scale.
```

Comment:
`millisecond delay` might be better?

Reply:
`single-digit millisecond performance` seems to come from Amazon DynamoDB.

Reply:
yes

Reply:
what about `delivers <10ms performance`?